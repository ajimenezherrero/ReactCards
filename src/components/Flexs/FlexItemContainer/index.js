import styled from 'styled-components';

import Flex from '../Flex';

const FlexItemContainer = styled(Flex)`
  flex: ${({ grow, shrink, basis }) => `${grow || 1} ${shrink || ''} ${basis || ''}`};
`;

export default FlexItemContainer;
