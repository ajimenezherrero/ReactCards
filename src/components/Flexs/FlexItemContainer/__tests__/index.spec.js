import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import FlexItemContainer from '../index';

describe('<FlexItemContainer />', () => {

  it('renders with its style', () => {
    const tree = renderer.create(<FlexItemContainer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
