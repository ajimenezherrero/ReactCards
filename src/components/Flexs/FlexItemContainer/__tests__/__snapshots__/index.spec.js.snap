// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlexItemContainer /> renders with its style 1`] = `
.c1 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
}

.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

<div
  className="c0 c1"
/>
`;
